src/zenml/integrations/deepchecks/__init__.py

```diff
@@ -36,21 +35,42 @@ class DeepchecksIntegration(Integration):
 
     NAME = DEEPCHECKS
     REQUIREMENTS = [
-        "deepchecks[vision]==0.8.0",
+        "deepchecks[vision]>=0.18.0",
         "torchvision>=0.14.0",
-        "pandas<2.0.0",
         "opencv-python==4.5.5.64",  # pin to same version
         "opencv-python-headless==4.5.5.64",  # pin to same version
         "tenacity!=8.4.0",  # https://github.com/jd/tenacity/issues/471
+        # TODO: Fix the explanation here
+        # Normally, the deepchecks integrations requires pandas to work.
+        # However, their version 0.8.0 is using a pandas function which got
+        # removed at pandas 2.1.0. Until we can upgrade the deepchecks
+        # requirement, we have to limit pandas to <2.1.0.
```

Comment:
Why did you have to upgrade deepchecks to >=0.18.0 then ?

Reply:
This was a leftover message which I now removed. I hope it is clear from the comment above why we had to upgrade the `deepchecks` dependency.